src/main/java/org/cascadebot/cascadebot/commands/informational/ServerInfoCommand.java

```diff
@@ -25,7 +25,12 @@ public void onCommand(Member sender, CommandContext context) {
         Guild guildForInfo = context.getGuild();
 
         if (context.getArgs().length > 0) {
-            guildForInfo = CascadeBot.INS.getShardManager().getGuildById(context.getArg(0));
+            try {
+                guildForInfo = CascadeBot.INS.getShardManager().getGuildById(context.getArg(0));
+            } catch (NumberFormatException e) {
```

Comment:
ummmm what? we're not doing any conversion to numbers here why do you have this

Reply:
getGuildById does actually throw number format exception if the input is not a long

Reply:
really? i though it just returned null

Reply:
Nope

Reply:
We really need better utils for this but I'm gonna look at that for beta